src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MsgPackLogExporter.cs

```diff
@@ -279,7 +276,7 @@ internal int SerializeLogRecord(LogRecord logRecord)
                 bodyPopulated = true;
                 continue;
             }
-            else if (this.m_customFields == null || this.m_customFields.ContainsKey(entry.Key))
+            else if (this.m_customFields == null || entry.Key == "name" || this.m_customFields.ContainsKey(entry.Key))
```

Comment:
@CodeBlanch @Yun-Ting I'm curious which one is correct / better?
1. entry.Key == "name"
2. string.Equals
3. string.Equals with Ordinal comparison

Reply:
It looks like customFields is using ordinal comparison: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/f1d19398539145fbd2621a86e31a5a2c033f95e8/src/OpenTelemetry.Exporter.Geneva/MsgPackExporter/MsgPackLogExporter.cs#L99

So might be nice to do string.Equals w/ ordinal to keep it consistent?

Reply:
Using string.Equals with Ordinal now